companion/gateway/utils/query_executor.py

```
@@ -0,0 +1,290 @@
+            # Determine query type
+            query_stripped = re.sub(r'/\*.*?\*/', '', query, flags=re.DOTALL)
+            query_stripped = re.sub(r'--.*?$', '', query_stripped, flags=re.MULTILINE)
+            query_stripped = query_stripped.strip().upper()
+
+            if query_stripped.startswith('SELECT'):
+                return await connection.fetch(query, *(params or ()))
+            elif query_stripped.startswith('INSERT') and 'RETURNING' in query_stripped:
+                return await connection.fetchrow(query, *(params or ()))
+            else:
+                return await connection.execute(query, *(params or ()))
+        except Exception as e:
```

Ruff (0.13.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)